src/access-control.module.ts

```diff
@@ -1,18 +1,27 @@
 import { Module, DynamicModule, Global, Abstract, Type } from '@nestjs/common';
+import { PATH_METADATA } from '@nestjs/common/constants';
 import { RolesBuilder } from './roles-builder.class';
 import { ROLES_BUILDER_TOKEN } from './constants';
+import { GrantsController } from './grants.controller';
 
 @Global()
 @Module({})
 export class AccessControlModule {
   /**
    * Register a pre-defined roles
    * @param {RolesBuilder} roles  A list containing the access grant
+   * @param {string} serveGrants A name of endpoint which serves grants
```

Comment:
I would prefer to name it `grantsEndpoint` instead of `serveGrants`.
since the `serveGrants` seems to imply a boolean type!